src/tests/topp/CMakeLists.txt

```diff
@@ -2866,7 +2866,7 @@ add_test("UTILS_ProteomicsLFQ_2" ${TOPP_BIN_PATH}/ProteomicsLFQ
          -out_cxml BSA_plusTransfer.consensusXML.tmp
          -out_msstats BSA_plusTransfer.csv.tmp
          -out BSA_plusTransfer.mzTab.tmp
-         -threads 4
+         -threads 1
```

Comment:
why?

Reply:
just because it does not really make a difference on the small dataset. Then more tests can be run in parallel.

Reply:
ok just wanted to make sure it is intended and not benchmarking related